price/domain/price.go

```diff
@@ -419,7 +424,7 @@ func (p Price) SplitInPayables(count int) ([]Price, error) {
 		if p.IsNegative() {
 			splittedAmount *= -1
 		}
-		prices[i] = NewFromInt(splittedAmount, precision, p.Currency())
+		prices[i] = NewFromInt(splittedAmount, precision, p.Currency()).GetPayable()
```

Comment:
Ich versteh nicht warum das notwendig ist? splittedAmount ist doch immer ein int und es dürfte keinen unterschied machen?

Reply:
The new price created by this NewFromInt does not create a payable.

You can try the test case "return payables".
Without this you get the correct sum up (which has been tested before only), but splitting up 88.39551 into 4 payables will result into 22.1, 22.1, 22.1, 22.09 if we don't apply GetPayable and return that.

As the payable representation for 88.39551 is 88.40 this imho fixes the issue.

We could also fix issues like this by calling GetPayable on every slice item we consume from the result of SplitInPayables.

Or - maybe a better solution:

we have a function Split which does not apply GetPayable on each item
and have the function SplitInPayables which has GetPayable applied on each item
to get it as flexible as possible.

Reply:
I checked the testcase - its green also when I remove the "..GetPayable()" ? for me

But I think if its the case that NewFromInt does not return a Payable price in some circumstances we should make sure it does and add the .GetPayable() to NewFromInt() ?

Reply:
Damn - I thought I pinned a combination where the error occurs. 

What do you think about having `Split` and `SplitInPayables` to have it as speaking as possible?

Maybe I also find the erratic calculation combination in my brain...

Reply:
I think the whole reason of the Split is to get Payables - I see no reason in offering another Split
(then you can just do Quo with the Big Int)

Reply:
mhm, indeed